### PR TITLE
Support CloudFormation Webhook

### PR DESCRIPTION
**Pull request by kirikiriyamama**
idobata

### CODE REVIEW

File: lib/hooks/cloudformation/helper.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+module Idobata::Hook
+  class Cloudformation
+    module Helper
+      def message
+        payload['Message'].gsub(/\n(?=')/, '').split("\n").each_with_object({}) do |s, h|
+          k, v = s.split('=')
+          h[k] = v.sub(/\A'/, '').sub(/'\z/, '')
+        end
```
Comment (ursm) [inline lib/hooks/cloudformation/helper.rb]: Is this easier to understand? (Umm...)

``` ruby
payload['Message'].gsub(/\n(?=')/, '').split("\n").map {|s|
  s.match(/\A(.+)='(.+)'\z/).to_a.drop(1)
}.to_h
```

File: lib/hooks/cloudformation/templates/subscription_confirmation.html.haml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+%p
+  = surround '', '.' do
+    You have chosen to subscribe to the topic
+    %b= payload.TopicArn
```
Comment (ursm) [inline lib/hooks/cloudformation/templates/subscription_confirmation.html.haml]:
``` haml
= succeed '.' do
  ...
```

Or this one may be preferred.

``` haml
%p You have chosen to subscribe to the topic <b>#{payload.TopicArn}</b>.
```

File: lib/hooks/cloudformation/helper.rb (hunk 1 above)
Comment (kirikiriyamama) [inline lib/hooks/cloudformation/helper.rb]: Your proposal seems the best. I'v applied it.

**Comment by ursm** [comment]
Thanks!

**Pull request merged**